apps/backend/src/Gump.Data/Exceptions/DocumentNotFoundException.cs

```diff
@@ -0,0 +1,24 @@
+using System.Runtime.Serialization;
+
+namespace Gump.Data.Repositories

```

Comment:
A namespace legyen tömb nélkül. Tehát ide egy pontosvesszőt, utána meg kivenni a kapcsos zárójeleket, ezzel minden sor egy kicsivel rövidebb lesz, ergó tisztább és olvashatóbb. Ha van máshol is ilyen, akkor ott is legyen fixálva.